From 90f56d62064a7dce6711e2b8547b5d6f99ad8011 Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Wed, 2 Jul 2025 11:09:45 +0200
Subject: [PATCH] arm64: dts: st: add power-domain property of csi in
 stm32mp215.dtsi

Add the power-domain property in the csi node.

Change-Id: I87bdae192fbcd76c2483e71fdf9353ca963d0392
Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/471698
Reviewed-by: Philippe CORNU <philippe.cornu@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Alain VOLMAT <alain.volmat@st.com>
Domain-Review: Philippe CORNU <philippe.cornu@foss.st.com>
Tested-by: Alain VOLMAT <alain.volmat@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp215.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp215.dtsi b/arch/arm64/boot/dts/st/stm32mp215.dtsi
index 7d2cbce154c9..077cb9a0e2eb 100644
--- a/arch/arm64/boot/dts/st/stm32mp215.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp215.dtsi
@@ -15,6 +15,7 @@ csi: csi@48020000 {
 			 <&rcc CK_KER_CSIPHY>;
 		clock-names = "pclk", "txesc", "csi2phy";
 		access-controllers = <&rifsc 86>;
+		power-domains = <&d1_pd>;
 		status = "disabled";
 	};
 };
-- 
2.53.0

